fodder.md

```diff
@@ -1 +1,3 @@
 * write about MHuot's Dev-Jam Attendee document
+* Jeff's talk at [Open Networking Summit 2016](http://opennetsummit.org/) is available to watch [on YouTube](https://youtu.be/IsPar4JPpec)
+* Jeff and Alejandro are hosting a drinkup in Houston, Texas on 06 April
```

Comment:
You can add a link to join the meet up http://www.meetup.com/OpenNMS-Meetup-Group/events/229919435/